lib/takeaway.rb

```diff
@@ -0,0 +1,52 @@
+class Takeaway
+
+  attr_reader :menu, :order_as_hash, :customer_supplied_total, :actual_total, :text
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)